build-dna.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/sh
```

i can't see any bash-only features here

Reply:
my previous suggestion seems to have not made it through :eyes: